cpu/cc26x0/cpu.c

```diff
@@ -42,21 +38,12 @@ void cpu_init(void)
     /* initialize the Cortex-M core */
     cortexm_init();
 
-    /* initialize the system clock */
-    cpu_clock_init();
+    /* Final trim of device */
```

Comment:
This also includes clock setup, right?

Reply:
Yes, in addition to preparing the micro to change the oscillator, it says: #13816

Reply:
Want to expand the comment a big?
This should then be good to go.